data/formats-data.js

```diff
@@ -870,7 +870,7 @@ exports.BattleFormatsData = {
 	},
 	electrode: {
 		randomBattleMoves: ["voltswitch","thunderbolt","taunt","foulplay","hiddenpowerice"],
-		randomDoubleBattleMoves: ["voltswitch","thunderbolt","taunt","foulplay","hiddenpowerice","protect","thunderwave"],
+		randomDoubleBattleMoves: ["voltswitch","discharge","taunt","foulplay","hiddenpowerice","protect","thunderwave"],
```

Comment:
Maybe it's better to code the check into the randbat set generator to reject Thunder Wave if you already have Discharge. Lanturn, Magnezone, and Porygon2 all have both on their sets as well, so this would be the better option.